resources/build/node.inc.sh

```diff
@@ -77,3 +77,7 @@ _node_select_version_with_nvm() {
     builder_die "Attempted to select node.js version $REQUIRED_NODE_VERSION but found $CURRENT_NODE_VERSION instead"
   fi
 }
+
+node_es_bundle() {
+  node "${KEYMAN_ROOT}/common/tools/es-bundling/build/common-bundle.mjs" "$@"
+}
```

Comment:
While refactoring, I consolidated all the `$BUNDLE_CMD` calls to use `node_es_bundle` instead:

1. DRY out lots of instances of `$BUNDLE_CMD` and `$bundle_cmd`
2. Use of a function instead of an unquoted variable is safer (e.g. space in pathnames) and more consistent with our other helper functions